src/overlays/actors/ovl_En_Zow/z_en_zow.h

```diff
@@ -7,9 +7,35 @@ struct EnZow;
 
 typedef void (*EnZowActionFunc)(struct EnZow*, GlobalContext*);
 
+#define ENZOW_GET_F(thisx) ((thisx)->params & 0xF)
+
+#define ENZOW_F_1 1
+
+typedef struct {
+    /* 0x00 */ u8 unk_00;
+    /* 0x04 */ f32 unk_04;
+    /* 0x08 */ f32 unk_08;
+    /* 0x0C */ UNK_TYPE1 unk_0C[0x3];
+    /* 0x0F */ u8 unk_0F;
+    /* 0x10 */ UNK_TYPE1 unk_10[0x4];
+    /* 0x14 */ Vec3f unk_14;
+    /* 0x20 */ Vec3f unk_20;
+    /* 0x2C */ Vec3f unk_2C;
+} EnZowStruct; // size = 0x38
+
 typedef struct EnZow {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x4D4];
+    /* 0x0144 */ SkelAnime skelAnime;
+    /* 0x0188 */ ColliderCylinder collider1;
```

Comment:
Why `collider1` instead of just `collider`?
```suggestion
    /* 0x0188 */ ColliderCylinder collider;
```